tests/pipelines/controlnet/test_controlnet_sdxl.py

```diff
@@ -998,7 +998,13 @@ def test_single_file_component_configs(self):
                 continue
             assert pipe.text_encoder_2.config.to_dict()[param_name] == param_value
 
-        PARAMS_TO_IGNORE = ["torch_dtype", "_name_or_path", "architectures", "_use_default_values"]
```

Comment:
maybe we should move all the slow tests for single_file into the same place - just so it is easy to have an overview of what kind of single format checkpoints we support

Reply:
Agreed